.pyre_configuration

```diff
@@ -48,6 +48,9 @@
     {
       "is_toplevel_module": true,
       "site-package": "typing_inspect"
+    },
+    {
+      "site-package": "tomli"
```

Comment:
`tomli` should be used only on Python < 3.11 because `tomllib` is in the Python standard library in Python >= 3.11.
* https://docs.python.org/3.11/whatsnew/3.11.html